### PR TITLE
update release guide and fix navigation error

### PR DESCRIPTION
**Pull request by liubao68**

### CODE REVIEW

File: script/release [hunk 1]
```diff
@@ -1,7 +1,28 @@
...
 
-git push origin asf-site
+## build source
+cd $CUR_DIR/servicecomb-website-master
+git checkout -f master
+bundle exec jekyll build
+
+## copy complied files
+cd $CUR_DIR/servicecomb-website-publish
+git checkout -B liubao -t origin/asf-site
```
Comment (WillemJiang) [inline script/release:18]: It can be changed to the website repo and the committer has the right to push the change.

File: script/release [hunk 2]
```diff
@@ -1,7 +1,28 @@
...
+git checkout -f master
+bundle exec jekyll build
+
+## copy complied files
+cd $CUR_DIR/servicecomb-website-publish
+git checkout -B liubao -t origin/asf-site 
+rm -R content/*
+mkdir content/references
+cp -R $CUR_DIR/servicecomb-website-master/content/* ./content/
+cp -R $CUR_DIR/servicecomb-java-chassis-doc/docs/* ./content/references/
```
Comment (WillemJiang) [inline script/release:22]: It's better to add links to the reference, as we have other docs need to link such as kie.

Comment (liubao68) [reply script/release:22]: Folder can not use symbolic link.  This way is the best option that can use now.

File: script/release (hunk 1 above)
Comment (liubao68) [reply script/release:18]: The push step is commented, and the publisher should use this step to commit changes and create a PR.

**Pull request merged**